src/azul/lambda_layer.py

```diff
@@ -73,16 +74,22 @@ class DependenciesLayer:
     lambda_dir = root / 'lambdas'
     reqs_file = root / 'requirements.txt'
     reqs_trans_file = root / 'requirements.trans.txt'
+    reqs_pip_file = root / 'requirements.pip.txt'
+    reqs_dev_file = root / 'requirements.dev.txt'
     wheel_dir = lambda_dir / '.wheels'
     layer_dir = lambda_dir / 'layer'
     out_dir = layer_dir / '.chalice' / 'terraform'
+    # We can force the layer to be rebuilt by changing this file. To do
+    # this predictably, increase the number in the next line monotonically.
+    update_counter = 0
```

Comment:
This should be the first attribute in the class because it is more likely going to be touched.